overlays/i2c1.dts

```diff
@@ -8,6 +8,7 @@
 
                 __overlay__ {
                         status = "okay";
+                        clock-frequency = < 0xf4240 >;
```

Comment:
We need to change frequency here?
https://github.com/iory/riberry/blob/087ed2dc6fbb4e57436d2ec4c8c4a336eea84582/bin/display_information.py#L92